### ISSUE TITLE
pagination does not work in tpcloud

### ISSUE THREAD

**Issue by arekusan** [winkm89/teachPress#68]
Hello,
in the shortcode tpcloud the pagination="1" brings up the pagination, but it doesn work on click. It allways shows the first page of the publication list. 
[tpcloud user="2" maxsize="20" minsize="9" tag_limit="100" pagination="1" entries_per_page="20" headline="3" style="numbered"]
Where is my mistake?
Thanks a lot
Arekusan

**Comment by winkm89** [comment]
Hi, 
Works for me in my test system. Could you post a link to your non working publication list?

**Comment by IlseO76** [comment]
Here's an example of pagination not working: https://www.esdit.nl/publications/scientific-publications. Links in the cloud do not work either.

**Comment by fabricebg** [comment]
> Here's an example of pagination not working: https://www.esdit.nl/publications/scientific-publications. Links in the cloud do not work either.

This *may* be due to the module "Divi", which seems to disable all links in the post. Try disabling it temporarily to see if this fixes the problem. If so, the problem comes from the theme, not teachPress.

**Comment by IlseO76** [comment]
@fabricebg: you are right, it was DIVI. Somehow Divi registers onClick events on the buttons which trigger some Divi Javascript (unknown what it is supposed to do). That Javascript prevented the browser to just request the URL that is tied to the Teachpress buttons/links. 

Solution: remove the onclick handler from the pagination buttons. Unfortunately the buttons have no specific Id or class, so we need to loop through all of them. 

For all other DIVI users out there - put this code in a code module on the page where you display the publication list: 

`<script>
jQuery(document).ready(function()
{
    // Loop through each numbers button element and remove the onclick handler
    jQuery(".tablenav-pages a.page-numbers.button").each(function()
    {
        jQuery(this).off("click");
    });
    // Do the same for all links in the Word cloud for the publications
    jQuery(".teachpress_cloud a").each(function()
    {
        jQuery(this).off("click");
    });
    // And, finally, also enable 'show all' button that clears filter from the word cloud
    jQuery(".teachpress_pub_list a").off("click");
});
</script>`
